### PR TITLE
removed divider from pages.

### PR DESCRIPTION
**Pull request by bigpoloskipainter**
Dividers removed
The divider from the following pages have been removed
-Staff
-Inspection Reports
-Gallery
-Football
-Hurling
-Basketball
-Choir
-School Calendar

### CODE REVIEW

**Comment by bigpoloskipainter** [comment]
@evanhearne please review

**Comment by evanhearne** [comment]
@bigpoloskipainter reviewing now.

**Comment by evanhearne** [comment]
@bigpoloskipainter when I said to delete the final div, I may not have explained myself correctly. The bottom div you deleted contained the code for the footer. I will quote the line that you should delete below. Please add back the footer div.

**Comment by evanhearne** [comment]
<div class="ui hidden section divider"></div>

This is the line I asked you to delete from above.

**Comment by evanhearne** [comment]
@bigpoloskipainter also Calendar.html has no divider, so you can just revert the changes you made on that file.

**Comment by laurafitzgerald** [comment]
@evanhearne is there a trello card for this change?

**Comment by evanhearne** [comment]
@laurafitzgerald yes there is. It just wasn't attached. I'll attach it now.

**Comment by laurafitzgerald** [comment]
Thanks @evanhearne. @bigpoloskipainter just a reminder that we have a template for pr descriptions which we should use to ensure we capture all the details around a change.

File: pages/about/Calendar.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
         </script>
     </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/about/Calendar.html]: @bigpoloskipainter this needs to be added back in.

Comment (evanhearne) [inline pages/about/Calendar.html]: @bigpoloskipainter this needs to be added back in.

File: pages/about/Calendar.html [hunk 2]
```diff
@@ -32,6 +32,6 @@ <h1 class="ui block header center blue aligned">School Calendar</h1>
 </div>
 </div>
 </div>
-    <div id="footer"></div>
```
Comment (evanhearne) [inline pages/about/Calendar.html]: @bigpoloskipainter and this.

File: pages/about/inspection.html [hunk 3]
```diff
@@ -17,7 +17,7 @@
   </script>
 </head>
 <body>
-  <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/about/inspection.html]: @bigpoloskipainter same here.

File: pages/about/inspection.html [hunk 4]
```diff
@@ -111,6 +111,6 @@ <h1 style="text-align: center;"><strong>W.S.E Report 2015</strong></h1>
       </div>
     </div>
   </div>
-  <div id="footer"></div>
```
Comment (evanhearne) [inline pages/about/inspection.html]: @bigpoloskipainter same here.

File: pages/about/staff.html [hunk 5]
```diff
@@ -17,7 +17,7 @@
         </script>
 </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/about/staff.html]: @bigpoloskipainter same here.

File: pages/about/staff.html [hunk 6]
```diff
@@ -72,6 +72,6 @@ <h3><strong>School Caretaker</strong></h3>
 </div>
         </div>
     </div>
-<div id="footer"></div>
```
Comment (evanhearne) [inline pages/about/staff.html]: @bigpoloskipainter same here.

File: pages/extra/basketball.html [hunk 7]
```diff
@@ -18,7 +18,7 @@
 
 </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/extra/basketball.html]: @bigpoloskipainter same here.

File: pages/extra/basketball.html [hunk 8]
```diff
@@ -30,6 +30,6 @@ <h2><strong>Basketball</strong></h2>
 </div>
         </div>
     </div>
-   <div id="footer"></div>
```
Comment (evanhearne) [inline pages/extra/basketball.html]: @bigpoloskipainter same here.

File: pages/gallery.html [hunk 9]
```diff
@@ -83,12 +82,12 @@ <h1 class="ui block header center blue aligned">Gallery</h1>
...
   <img data-src="https://lh3.googleusercontent.com/IfWXpXkdUxsfKjn85Lska77Zq1W8PlMzYRWJhqzRAyP5ItZ6g5u-78FtblUgORVVM763V235trv8WXUJrttfrN-rsEwrr__8E0z0MRrY5_DJnipTxrizPOChfqLv3EKi7c29sT928w=w1920-h1080" src="" alt="" />
   <img data-src="https://lh3.googleusercontent.com/2uk3nHVxyp6EZEmd-UNeDr0mq7isMu3iwG4z8HZNrZcKNrvBtWBzfZ02yZYRei-zfSaPFOvwwV_e8CaDbjhPXtUkmb6_uAfqR-K6mfBO05loxtRfQkmJprGFgXzMDulFUPq5TTYNsg=w1920-h1080" src="" alt="" />
-</div>
+
 
               
             </div>
         </div>
     </div>
-    <div id="footer"></div>
```
Comment (evanhearne) [inline pages/gallery.html]: @bigpoloskipainter same here.

File: pages/gallery.html [hunk 10]
```diff
@@ -18,15 +18,14 @@
...
-    <div class="ui divbg container">
-        <img class="ui centered image" height="128px" src="../assets/images/logo.jpg">
+            <img class="ui centered image" height="128px" src="../assets/images/logo.jpg">
         <div id="navbar"></div>
         <div class="ui padded basic segment">
             <h1 class="ui block header center blue aligned">Gallery</h1>
             <div class="ui hidden section divider"></div>
             <div class="ui blue center aligned segment">
               <script src="https://cdn.jsdelivr.net/npm/publicalbum@latest/dist/pa-embed-player.min.js" async></script>
-<div class="pa-embed-player" style="width:100%; height:480px; display:none;"
```
Comment (evanhearne) [inline pages/gallery.html]: @bigpoloskipainter why was this removed?

File: pages/gallery.html [hunk 11]
```diff
@@ -83,12 +82,12 @@ <h1 class="ui block header center blue aligned">Gallery</h1>
   <img data-src="https://lh3.googleusercontent.com/X97n2_5Edhcw7U5-y9kIzYygKM7t_Gnp4njezQrK6t8fAWG9mrIxZpXgC_goDR35Yqm63tROU_oW9zjiT87uJ8JbYBCyfLXvTmLjoVpL0Soy3rvc8PThG1jF546dhvphDDRVQ_2NHA=w1920-h1080" src="" alt="" />
   <img data-src="https://lh3.googleusercontent.com/IfWXpXkdUxsfKjn85Lska77Zq1W8PlMzYRWJhqzRAyP5ItZ6g5u-78FtblUgORVVM763V235trv8WXUJrttfrN-rsEwrr__8E0z0MRrY5_DJnipTxrizPOChfqLv3EKi7c29sT928w=w1920-h1080" src="" alt="" />
   <img data-src="https://lh3.googleusercontent.com/2uk3nHVxyp6EZEmd-UNeDr0mq7isMu3iwG4z8HZNrZcKNrvBtWBzfZ02yZYRei-zfSaPFOvwwV_e8CaDbjhPXtUkmb6_uAfqR-K6mfBO05loxtRfQkmJprGFgXzMDulFUPq5TTYNsg=w1920-h1080" src="" alt="" />
-</div>
```
Comment (evanhearne) [inline pages/gallery.html]: @bigpoloskipainter same here.

File: pages/extra/hurling.html [hunk 12]
```diff
@@ -30,6 +30,6 @@ <h2><strong>Hurling</strong></h2>
 </p>
         </div>
     </div>
-    <div id="footer"></div>
```
Comment (evanhearne) [inline pages/extra/hurling.html]: @bigpoloskipainter same here.

File: pages/extra/hurling.html [hunk 13]
```diff
@@ -17,7 +17,7 @@
         </script>
 </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/extra/hurling.html]: @bigpoloskipainter same here.

File: pages/extra/football.html [hunk 14]
```diff
@@ -17,7 +17,7 @@
         </script>
 </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/extra/football.html]: @bigpoloskipainter same here.

File: pages/extra/football.html [hunk 15]
```diff
@@ -29,6 +29,6 @@ <h2><strong>Football</strong></h2>
 </div>
         </div>
     </div>
-    <div id="footer"></div>
```
Comment (evanhearne) [inline pages/extra/football.html]: @bigpoloskipainter same here.

File: pages/extra/choir.html [hunk 16]
```diff
@@ -29,6 +29,6 @@ <h2><strong>Mount Sion Choir</strong></h2>
 </div>
         </div>
     </div>
-    <div id="footer"></div>
```
Comment (evanhearne) [inline pages/extra/choir.html]: @bigpoloskipainter same here.

File: pages/extra/choir.html [hunk 17]
```diff
@@ -17,7 +17,7 @@
         </script>
 </head>
 <body>
-    <div class="ui divbg container">
```
Comment (evanhearne) [inline pages/extra/choir.html]: @bigpoloskipainter same here.

**Comment by laurafitzgerald** [comment]
@evanhearne can you give @bigpoloskipainter a hand with this if he needs it.

**Comment by evanhearne** [comment]
@laurafitzgerald I will.

**Comment by evanhearne** [comment]
@bigpoloskipainter im adding one change as discussed, after this please look at the trello card attached and redo this task.

**Comment by evanhearne** [comment]
@bigpoloskipainter you can proceed with your task. :)

**Comment by evanhearne** [comment]
@bigpoloskipainter changes look good, but because you pulled after changes it added the divider back to gallery.html. Please make these changes and push them up. :)

**Comment by evanhearne** [comment]
@bigpoloskipainter all changes are fine, all you need to do is resolve conflicts. There is a guide [here](https://github.com/MountSionCBSSecondary/SchoolWebsite/blob/master/docs/Tutorial.md) under Rebasing. Lemme know if you need help.

**Comment by evanhearne** [comment]
@laurafitzgerald that sounds like a good idea. @bigpoloskipainter when you are in next, there is a guide on how to squash commits [here](https://github.com/wprig/wprig/wiki/How-to-squash-commits)

**Comment by evanhearne** [comment]
@laurafitzgerald can you please re-review this when you have the time?

**Comment by evanhearne** [comment]
@laurafitzgerald can you please re-review?

File: pages/gallery.html [hunk 18]
```diff
@@ -18,18 +18,20 @@
...
         <div class="ui padded basic segment">
             <h1 class="ui block header center blue aligned">Gallery</h1>
-            <div class="ui hidden section divider"></div>
             <div class="ui blue center aligned segment">
               <script src="https://cdn.jsdelivr.net/npm/publicalbum@latest/dist/pa-embed-player.min.js" async></script>
-              <script src="https://cdn.jsdelivr.net/npm/publicalbum@latest/dist/pa-embed-player.min.js" async></script>
 <div class="pa-embed-player" style="width:100%; height:480px; display:none;"
   data-link="https://photos.app.goo.gl/U7HVjKDWjXeACoeo9"
   data-title="Gallery"
+<<<<<<< HEAD
```
Comment (laurafitzgerald) [inline pages/gallery.html]: Rebase code still here.

**Comment by evanhearne** [comment]
@laurafitzgerald review when you have the time. Changes have been done.
